TwitterTweetSentimentalAnalysis/Model.py

```diff
@@ -0,0 +1,87 @@
+# this is a local Credential Helper
+import tweepy
+from nltk.sentiment import SentimentIntensityAnalyzer
+from textblob import TextBlob
+
+import TwitterTweetSentimentalAnalysis.CredentialHelper as twitterCredential
+
+
+# Tweepy is a lib that can be used to connect to twitter api
+# helps us to make sentimental analysis on the given data on bases of text
+
+
+class model(object):
+
+    def __init__(self):
+        super().__init__()
+
+    @staticmethod
+    def get_authenticated_api():
+        auth = tweepy.OAuthHandler(twitterCredential.candidate_key, twitterCredential.candidate_sec)
+        auth.set_access_token(twitterCredential.access_key, twitterCredential.access_sec)
+        api = tweepy.API(auth)
+        return api
+
+    def get_user_name(self):
+        api = self.get_authenticated_api()
+        user = api.me()
+        return user.name
+
+    def get_live_tweets_from_Twitter(self, text):
+        api = self.get_authenticated_api()
+        # getting the tweets form twitter using The Twitter aap and tweepy
+        # Demo of getting text from twitter
+
+        tweet_live = api.search(text, tweet_mode='extended')
+        # for tweet in tweet_live:
```

Comment:
remove these comments from the whole code, textual comments are fine but remove the code comments